container-search/src/main/java/com/yahoo/search/query/ranking/Matching.java

```diff
@@ -52,6 +54,7 @@ public class Matching implements Cloneable {
     private Double postFilterThreshold = null;
     private Double approximateThreshold = null;
     private Double targetHitsMaxAdjustmentFactor = null;
+    private Double filterThreshold;
```

Comment:
Consider initializing `filterThreshold` to `null`, as in the lines above.

Reply:
Fixed